src/test/ui/wf/wf-fn-where-clause.rs

```diff
@@ -13,5 +13,8 @@ fn bar() where Vec<dyn Copy>:, {}
 //~^ ERROR E0277
 //~| ERROR E0038
 
+struct Vec<T> {
+    t: T,
+}
```

Comment:
Can you keep the original test and duplicate it with a local `Vec` definition?
Ideally the original test would also have good diagnostics.

Reply:
The tests where I did this have good output, they just point into stdlib when their spans are available and changing them to a local type didn't compromise the integrity of what was being tested. It didn't feel necessary to have two types (increasing CI time), but if you insist I can duplicate them.

Reply:
> necessary to have two types (increasing CI time)

Presumably it takes negligible time, it's only 3 tests.

> and changing them to a local type didn't compromise the integrity of what was being tested

Can never be too sure, especially for old tests.